### PR TITLE
Update Thamos and provide requirements.in files

### PR DESCRIPTION
**Pull request by fridex**
## This introduces a breaking change

- [x] No

### CODE REVIEW

File: ubi8-py36/requirements.txt [hunk 1]
```diff
@@ -40,78 +42,121 @@ aiohttp==3.7.3; python_version >= '3.6' \
...
     --hash=sha256:9f47eda37229f68eee03b24b9748937c7dc3868f906e8ba69fbcbdd3bc5dc3e2
+    # via rich
 commonmark==0.9.1 \
     --hash=sha256:452f9dc859be7f06631ddcb328b6919c67984aca654e5fefb3914d54691aed60 \
     --hash=sha256:da2f38c92590f83de410ba1a3cbceafbc74fee9def35f9251ba9a971d6d66fd9
+    # via rich
 daiquiri==3.0.0 \
     --hash=sha256:1df8f03a9a59d82108d7fc4c0e531f0d049191859eab3a6e22df13b46bf123f8 \
     --hash=sha256:cbc4a1ffdd98f25bb1962f3d580d5225796196712849798cce40af85c412d5ab
-dataclasses==0.8; python_version >= '3.6' and python_version < '3.7' \
```
Comment (harshad16) [inline ubi8-py36/requirements.txt:80]: this package is removed , this package was required on the previous version for thamos

**Comment by fridex** [comment]
/assign @harshad16 

Checked environment markers, it looks like they are not produced. Briefly checking all should be fine, we can try to include it in the next release.

**Comment by harshad16** [comment]
> Checked environment markers, it looks like they are not produced. Briefly checking all should be fine, we can try to include it in the next release.

in the python 3.6 the package dataclasses and idna-ssl were removed.
which has previously caused issues.

**Comment by fridex** [comment]
> > Checked environment markers, it looks like they are not produced. Briefly checking all should be fine, we can try to include it in the next release.
> 
> in the python 3.6 the package dataclasses and idna-ssl were removed.
> which has previously caused issues.

Ah, good catch. Please take a look at https://github.com/thoth-station/s2i-thoth/pull/139, it should have the fix for specific Python version.

**Pull request closed without merging**